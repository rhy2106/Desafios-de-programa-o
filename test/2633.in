3
picanha 15
coracao 14
maminha 37
4
alcatra 17
linguica 13
asinha 5
pernil 23
5
a 9
b 2
c 3
d 5
e 1