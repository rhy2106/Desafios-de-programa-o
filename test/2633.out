coracao picanha maminha
asinha linguica alcatra pernil
e b c d a
